.gitignore

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
we're deliberately not including .gitignore.  none of the other dunedaq repos have it, and i don't see a reason for this one to differ.

Reply:
This still needs to be removed from the PR.